Makefile

```diff
@@ -114,35 +108,36 @@ pytest: compose-build
 	@docker-compose run $(IMAGE_NAME) \
 		pytest $(pytestpath) -m "not integration and not integration_erasure and not integration_external"
 
-# Run the pytest integration tests.
-pytest-integration-access: compose-build
-	@echo "Building additional Docker images for integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml build
-	@echo "Bringing up the integration environment..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml up -d
-	@echo "Waiting 20s for integration containers to be ready..."
-	@sleep 20
-	@echo "Running additional setup for mssql integration tests"
-	@docker exec fidesops python tests/integration_tests/mssql_setup.py
-	@echo "Running pytest integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml \
-		run $(IMAGE_NAME) \
-		pytest $(pytestpath) -m integration
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml down --remove-orphans
+configure-infrastructure: compose-build
+	@echo "creating virtualenv"
+	@virtualenv -p python3 fidesops_test_dispatch
+	@echo "activating virutalenv"; \
+		source fidesops_test_dispatch/bin/activate; \
+		python configure_infrastructure.py datastores=$(datastores)
+
+
+pytest-integration: compose-build
+	@virtualenv -p python3 fidesops_test_dispatch; \
+		source fidesops_test_dispatch/bin/activate; \
+		python configure_infrastructure.py run_tests=true datastores=$(datastores)
+
 
 pytest-integration-erasure: compose-build
 	@echo "Building additional Docker images for integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml build
-	@echo "Bringing up the integration environment..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml up -d
-	@echo "Waiting 20s for integration containers to be ready..."
-	@sleep 20
-	@echo "Running additional setup for mssql integration tests"
-	@docker exec fidesops python tests/integration_tests/mssql_setup.py
-	@echo "Running pytest integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml \
-		run $(IMAGE_NAME) \
-		pytest $(pytestpath) -m "integration_erasure"
+	@path=""; \
```

Comment:
why can't we re-use the `configure_infrastructure.py` for erasures?

Reply:
We can, this is due to be handled in a subsequent ticket. The main point here is that we need to make sure we're creating test fixtures properly before we can handle erasures properly. When we handle fixture creation in the right way, we no longer need a delineation for erasure tests and they can be run at the same time as access requests (because access requests won't fail if the erasure test has run first).

Reply:
ah I see, perfect!